rosbag2_transport/include/rosbag2_transport/player.hpp

```diff
@@ -381,6 +381,7 @@ class Player : public rclcpp::Node
 
   /// \brief Getter for the first of the currently stored storage options
   /// \return Reference to the first item in the StorageOptions vector
+  [[deprecated("Use rosabg2_transport::Player::get_all_storage_options() instead")]]
```

Comment:
we usually do not do this for the already-released distribution in the policy of https://docs.ros.org/en/rolling/The-ROS2-Project/Contributing/Developer-Guide.html#deprecation-strategy? as you mentioned, this is protected method, but could be used by the application already? in that case, we are breaking our own rule... i am not saying we never do this, but would like to know how others feel... 🤔 @christophebedard wdyt?

Reply:
I don't have a strong opinion, but I would err on the side of caution and simply not deprecate it.

Reply:
I can remove the deprecation. However, I wouldn't print an error since the method is functional as before.  
I put a deprecation notice as a fair warning not to use it since we will remove it sometime anyway, and an alternative is already present in the target distro.

Reply:
Okay. I removed the deprecation warning.  
However, I don't think that we will need to print an error if we have more than one storage option.
The Doxygen comment pretty much clears what this function does and what would be the expected behavior 
```cpp
  /// \brief Getter for the first of the currently stored storage options
  /// \return Reference to the first item in the StorageOptions vector
```
and behavior is consistent with expectations.